sdk/aqueduct/models/dag.py

```diff
@@ -28,6 +28,7 @@ class Schedule(BaseModel):
     trigger: Optional[TriggerType] = None
     cron_schedule: str = ""
     disable_manual_trigger: bool = False
+    source_id: Optional[uuid.UUID] = None
```

Comment:
Can you add a comment here describing what this represents, and that it is only relevant for a specific trigger type?